### DISCUSSION TITLE
Hover/Lift off distance

### DISCUSSION THREAD

**Discussion by g-y617** [OpenTabletDriver/OpenTabletDriver#1317] [Q&A] [answered]
Is it possible to change the lift off distance of a tablet using OTD? It's pretty annoying to have to lift my pen 2 cm up, especially when I use relative mode. I see in the tablet debugger that it detects the hover distance, so it would make sense to be able to change it.

**Comment by InfinityGhost** [comment, marked as answer]
It could be done in v0.6 with a custom filter plugin, but it won't be implemented as a feature into OpenTabletDriver.

**Reply by jamesbt365** [reply]
there is now a hover limiter plugin.

**Reply by Maxlols** [reply]
i neeedd it in 5.3.3 cuz im unable to use my tablet and pen on valo using the newer versions
